### ISSUE TITLE
Two questions

### ISSUE THREAD

**Issue by wjczf123** [AtmaHou/FewShotJoint#3]
Hi, i have two questions about this paper.
1. The baseline LD-proto uses logits-dependency tricks to enhance the performance. What is logits-dependency tricks? I only find slot-gate in Goo et al., 2018.
2. I notice there is difference between FewJoint 1.0 and FewJoint 1.1. Does this mean that FewJoint 1.1 is more suitable for the benchmark dataset?

**Comment by AtmaHou** [comment]
1. For logits-dependency tricks, we get the correspondence between intents and slots based on the support set, and then add the logits of corresponding intents to the logits of slots when predicting slots, and vice versa for intent.
2. Yes.

**Comment by AtmaHou** [comment]
1. For logits-dependency tricks, we get the correspondence between intents and slots based on the support set, and then add the logits of corresponding intents to the logits of slots when predicting slots, and vice versa for intent.
2. Yes.

**Comment by wjczf123** [comment]
OK. Thank you a lot.
